packages/graphback-core/src/scalars/objectId.ts

```diff
@@ -0,0 +1,27 @@
+import { ObjectID as BsonObjectID } from "bson";
+
+/* eslint-disable */
```

Comment:
Could a function scoped blanket disablement of ESLint mask other warnings in future?

Reply:
Theoretically yes, but I honestly do not see us adding anything else in these two functions.

Reply:
We probably won't yes, but there is a possibility that if our ESLint rules change and this function is affected it won't be noticed. Unlikely scenario anyway.